app/models/hydrus.rb

```diff
@@ -1,3 +1,2 @@
 module Hydrus
-
```

Comment:
This module has to be garbage, right? The `Hydrus` module gets defined over in `config/application.rb`.